### ISSUE TITLE
the v2t and t2v

### ISSUE THREAD

**Issue by Arsiuuu** [Ziyang412/UCoFiA#8]
Sorry to bother you again. I'm confused about this code. 

https://github.com/Ziyang412/UCoFiA/blob/6098be26627e445f03de5dc0f4b1db74f55d39d3/train/modules/modeling_ucofia.py#L349C9-L354C1

1. The first matrix calculates the similarity from sentence to video(t2v), while the second matrix seems to calculate the similarity from frame to sentence(v2t) because the .t() in the end. Is it allowed to directly sum these two matrices?
2. One more thing, the third matrix seems to be bidirectional,  does it have different effects on v2t and t2v?

Thanks for your attention!

**Comment by Ziyang412** [comment]
Thanks for asking.

1. I think the  .t() operation is just matching the matrix dimension, I think the first logits also have .t() operation
2. I am a bit confused by the "Is it allowed to directly sum these two matrices", maybe you could provide more details. A general response is this is for match the video-text pair, there are not t2v or v2t setting consideration in this stage (that one is about inferencing). 
3. Similar to 2, the goal is just compute the similarity given the vid-txt pair, please refer to eval function/file for more detail on v2t or t2v.

Thanks
